ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -66,8 +66,8 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
-- as: aws-ipi-disconnected-private-f28-destructive
-  cron: 28 2 14 * *
+- as: aws-ipi-disc-amd-mixarch-f28-destructive
```

Comment:
Do we intend to skip `private` in the name?

Reply:
Right, on aws, all the disconnected cluster is combining private feature, so I remove it directly to save some chars.